subworkflows/local/prepare_genome.nf

```diff
@@ -155,14 +155,15 @@ workflow PREPARE_GENOME {
     }
 
     emit:
-    fasta         = ch_fasta                  //    path: genome.fasta
-    gtf           = ch_gtf                    //    path: genome.gtf
-    gene_bed      = ch_gene_bed               //    path: gene.bed
-    tss_bed       = ch_tss_bed                //    path: tss.bed
-    chrom_sizes   = ch_chrom_sizes            //    path: genome.sizes
-    filtered_bed  = ch_genome_filtered_bed    //    path: *.include_regions.bed
-    bwa_index     = ch_bwa_index              //    path: bwa/index/
-    autosomes     = ch_genome_autosomes       //    path: *.autosomes.txt
+    fasta         = ch_fasta                      //    path: genome.fasta
+    fai           = CUSTOM_GETCHROMSIZES.out.fai  //    path: genome.fai
```

Comment:
I will create a ch_fai channel for consistency just below this line ```ch_chrom_sizes = CUSTOM_GETCHROMSIZES ( ch_fasta ).sizes``` i.e. `ch_fai = CUSTOM_GETCHROMSIZES ( ch_fasta ).fai`